packages/react-composites/src/composites/CallComposite/components/LobbyTile.tsx

```diff
@@ -70,6 +70,7 @@ export const LobbyTile = (props: LobbyTileProps): JSX.Element => {
               className={mergeStyles(titleStyle(palette, isVideoReady))}
               aria-live="assertive"
               data-ui-id="lobbyScreenTitle"
+              role="alert"
```

Comment:
We should probably add this to the notification component also... 👀 https://github.com/Azure/communication-ui-library/blob/98498c85eb6d6c9296e2406538650023af8de200/packages/react-components/src/components/Notification.tsx#L183

Reply:
May need it in the config message bars:

(beta)
https://github.com/Azure/communication-ui-library/blob/98498c85eb6d6c9296e2406538650023af8de200/packages/react-components/src/components/TroubleshootingGuideErrorBar.tsx#L116

https://github.com/Azure/communication-ui-library/blob/98498c85eb6d6c9296e2406538650023af8de200/packages/react-components/src/components/TroubleshootingGuideErrorBar.tsx#L133

(stable)
https://github.com/Azure/communication-ui-library/blob/98498c85eb6d6c9296e2406538650023af8de200/packages/react-composites/src/composites/CallComposite/components/ConfigurationPageErrorBar.tsx#L82

Reply:
I don't know about adding alerts into the notifications and error bar for now... i think we punt it and see what remains for now